lib/update_school_data.rb

```diff
@@ -78,7 +78,10 @@ def set_simple_properties(school, row)
   end
 
   def set_gias_data_as_json(school, row)
-    school.gias_data = row.to_json
+    scratch = {}
```

Comment:
I'm not sure where the word 'scratch' came from?

Reply:
'scratch' as in 'temporary'.